free-2.md

```diff
@@ -1,44 +1,77 @@
 # Free project 2
 
-
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Recipes
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Right now, recipes can range from overly informative ("use a 3.9 inch pan and 
+fry for precisely 2 minutes and 7.3 seconds") to not nearly informative enough
+("fry the food").  This can cause difficulty for the amateur chef - the devil is
+in the details and the details aren't there.  There experience could be improved
```

Comment:
It can be.  If I saw a recipe like the first one I either wouldn't do it (because I was worried those precise steps were needed) or I'd ignore most of the instructions (defeating the purpose).